.github/workflows/release-notes.yml

```diff
@@ -67,7 +67,8 @@ jobs:
       # is not included in the GitReleaseManager action yet.
       - name: Generate Change Log
         run: |
-          dotnet-gitreleasemanager export --token ${{ secrets.PERSONAL_ACCESS_TOKEN }} -o 'corgibytes' -r 'freshli-lib' -f 'CHANGELOG.md'
+          dotnet-gitreleasemanager export --token ${{ secrets.GITHUB_TOKEN }} -o 'corgibytes' -r 'freshli-lib' -f 'CHANGELOG.md'
+          git add --renormalize CHANGELOG.md
```

Comment:
> Thank you for fixing the end of line issue. I think the CLA check is failing for two reasons:
> 
> 1. It still points to the "freshli" repo instead of "freshli-lib".
> 2. You appear to be using two separate GitHub accounts for this PR.  My guess is you have one account setup in your Git Client and the 2nd one when you login to the GitHub website.  It's probably best if you stuck with one GitHub account for all your actions.
> 
> The short version is I think the CLA check can be ignored for this PR.

Yes, I noticed that I am using different git accounts. The corgi one and my personal one. Will fix that. hehe.